tests/Assets/Entities/MissingSource.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tests\Metabolize\Assets\Entities;
+
+use DateTimeImmutable;
+use Minormous\Metabolize\Dali\Attributes\Column;
+use Minormous\Metabolize\Dali\Attributes\Source;
+use Minormous\Metabolize\Dali\Attributes\Table;
+
+#[Table('test')]
+class MissingSource
+{
+    #[Column(isIdentifier: true)]
```

Comment:
Should Identifier be an annotation on its own?

Reply:
It could be, if that is what we wanted.

In the old Metabolize, it is an option on Column, and because it is a type of column, I would think makes sense to have isIdentifier on the Column attribute.

I don't know, it could be I suppose.